packages/comment-to-assert/src/ast-utils.ts

```diff
@@ -153,5 +153,4 @@ export function wrapAssert(
             EXPECTED_NODE,
         });
     }
-    throw new Error("Unknown pattern: " + actualNode);
```

Comment:
Instead of this, `actualNode satisfies never` is just pattern.
It will make compiler error when reach unknown type.